tests/testutils/ctx.go

```diff
@@ -144,6 +144,12 @@ func (ctx *RktRunCtx) Reset() {
 
 func (ctx *RktRunCtx) cleanupChildren() error {
 	for _, child := range ctx.children {
+		if child.Cmd == nil ||
+			child.Cmd.Process == nil ||
+			child.Cmd.ProcessState == nil {
+			continue
```

Comment:
If the child isn't fully initialized there might be a panic due to nil-pointer. I ran into this while testing locally and I think the cleanup function should be able to tolerate this.